src/main/java/org/springframework/data/couchbase/core/CouchbaseOperations.java

```diff
@@ -50,4 +54,9 @@ public interface CouchbaseOperations extends FluentCouchbaseOperations {
 	 * Returns the default consistency to use for queries
 	 */
 	QueryScanConsistency getConsistency();
+
+	<T> T save(T entity, String... scopeAndCollection);
```

Comment:
scopeAndCollection as a string seems a bit weird, we never have that concept somewhere else do we?

Reply:
if it's internal mark as internal and maybe expose it with a CollectionIdentifier instead?